repository_service_tuf/cli/admin/import_targets.py

```diff
@@ -56,7 +67,8 @@ def _parse_csv_data(
 
 def _import_csv_to_rstuf(
     db_client: Any,
```

Comment:
Can you improve the annotation to be more precise?
From what I see `db_client` is a `sqlaclehmy` object called `Connection`.

Reply:
Oh, I notice something.

If I do it, I will need to import Connection and Table outside the CLI command.
The SQLAlchemy will become a required [instead of an optional dependency](https://repository-service-tuf.readthedocs.io/en/v1.0.0a1-draft/guide/repository-service-tuf-cli/index.html#import-targets-import-targets).